pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -1096,71 +1096,72 @@ func (m *kubeGenericRuntimeManager) computeInitContainerActions(pod *v1.Pod, pod
 				break
 			}
 
-			if podutil.IsRestartableInitContainer(container) {
```

Comment:
can we keep this if statement to see a clear diff for this PR?

I understand this line is unnecessary because of L1096, but it makes the diff hard to see.

Reply:
@gjkim42 on github you can just check the `Hide Whitespace` setting for the diff

But either way, I'm ok with leaving this.